samtools.wdl

```diff
@@ -57,6 +57,46 @@ task BgzipAndIndex {
     }
 }
 
+task Faidx {
+    input {
+        File inputFile
+        String outputDir
+        String basenameInputFile = basename(inputFile)
+
+        String memory = "2G"
+        String dockerImage = "quay.io/biocontainers/samtools:1.10--h9402c20_2"
+    }
+
+    command <<<
+        set -e
+        mkdir -p "$(dirname ~{outputDir})"
+        ln -s ~{inputFile} "~{outputDir}/~{basenameInputFile}"
```

Comment:
Just hard-link it with `ln` that will work better I think. But maybe @DavyCats has a different experience?